File: week08/15/1_Mon_TieRope.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(K, A):
    # Implement your solution here
    count = 0
    length = 0
    for rope in A:
        length += rope
        if length >= K:
            count += 1
            length = 0
    return count

File: week08/15/1_Mon_TieRope.cpp
// you can use includes, for example:
// #include <algorithm>
#include <vector>
using namespace std;
// you can write to stdout for debugging purposes, e.g.
// cout << "this is a debug message" << endl;

int solution(int K, vector<int> &A) {
    // Implement your solution here
    int count = 0;
        int length = 0;
        for(int rope : A) {
            length += rope;
            if (length >= K) { 
                count++;
                length = 0;
            }
        }
        return count;
}